code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -366,16 +366,15 @@
 //This proc is called when you want to place an item into the storage item.
 /obj/item/weapon/storage/attackby(obj/item/W, mob/user, params)
 	..()
-
+	. = 1 //no afterattack

```

Comment:
so i cant c4 my bag full of c4 when i try to put the c4 in the bag but the bag is full of c4 so it plants the c4 on the bag of c4 instead?